src/Auth/sagas.ts

```diff
@@ -86,7 +86,16 @@ function* doAuth() {
 
 function* logout() {
   const logoutEndpoint: string = yield select(selectors.getLogoutEndpoint);
-  window.location.href = logoutEndpoint;
+  const loginProvider: string = yield select(selectors.getLoginProvider);
+  if (loginProvider) {
+    fetch(logoutEndpoint, { headers: { accept: 'application/json' } })
+      .then((response) => response.json())
+      .then((jsonResponse) => {
+        window.location.assign(`${jsonResponse.logout_url}&return_to=${window.location.href}`);
+      });
+  } else {
+    window.location.assign(`${logoutEndpoint}?return_to=${window.location.href}`);
+  }
```

Comment:
Small note - this implementation differs from the other portals in the following details:
- Here the 2 step logout process will only happen if `LOGIN_PROVIDER` is set.
- The other portals pass `return_to` to the fetch.

Reply:
You right. I don't see any reason this implementation be different from others. I refactored to the other portal standard.